packages/shared-ux/page/solution_nav/src/solution_nav.tsx

```diff
@@ -177,7 +179,7 @@ export const SolutionNav: FC<SolutionNavProps> = ({
     );
   }, [children, headingID, isCustomSideNav, isHidden, items, rest]);
 
-  const { euiTheme } = useEuiTheme();
+  const { euiTheme } = euiThemeContext;
```

Comment:
Nit: wouldn't this be simpler: `const { euiTheme } = useEuiTheme();` I'm not seeing why we need the added `euiThemeContext` variable declared on L106.

Reply:
You're totally right - this was a change I forgot to revert from when I was trying out a different approach (that ended up failing). Thanks for catching it Tim! 🙏

Reply:
https://github.com/elastic/kibana/pull/177361/commits/a13b2c7a0b2b37e2bcb9fb203463b246d5b6dae4